io-backend/src/main/java/agh/io/iobackend/service/GameService.java

```diff
@@ -15,12 +22,30 @@ public GameService() {
         this.games = new HashMap<>();
     }
 
+    public void createGame(GameRoomRequest gameRoomRequest, Long gameId){
+        // na razie podaje gameRoomId jako room id i game id
+        GameState gameState = new GameState(gameId, gameId, gameRoomRequest.getMapId(), gameRoomRequest.getGameMasterId());
```

Comment:
Czy to nie jest błąd, że w konstruktorze `GameState()` `gameRoomId` jest takie samo jak `gameId`? Domyślam się, że po prostu teraz tak przyjmujemy, ale proszę o potwierdzenie tego przypuszczenia.